opentelemetry/proto/profiles/v1experimental/pprofextended.proto

```diff
@@ -77,6 +77,8 @@ message Profile {
   repeated Sample sample = 2;
   // Mapping from address ranges to the image/binary/library mapped
   // into that address range.  mapping[0] will be the main binary.
+  // If multiple binaries contribute to the Profile and no main
+  // binary can be identified, mapping is optional.
```

Comment:
I think maybe just rephrase "mapping is optional", I don't think we're saying the entire field should be skipped, we're saying that the 0th item has no special meaning, right? At least that's how I understood yesterday's conversation.

Reply:
I have rephrased it from `[..], mapping is optional.` to `[..], mapping[0] has no special meaning.` with https://github.com/open-telemetry/opentelemetry-proto/pull/556/commits/00ea0e1e6a012da16e810a87615164d047991969. Hope this is fine.